addons/arsenal/functions/fnc_removeBox.sqf

```diff
@@ -42,8 +42,13 @@ if (_global && {isMultiplayer} && {!isNil "_id"}) then {
 };
 
 // If the arsenal is already open and not ignoring content (see FUNC(openBox)), close arsenal display
-if (!isNil QGVAR(currentBox) && {GVAR(currentBox) isEqualTo _object} && {isNil QGVAR(ignoredVirtualItems)}) then {
-    [LLSTRING(noVirtualItems), false, 5, 1] call EFUNC(common,displayText);
-    // Delay a frame in case this is running on display open
-    [{(findDisplay IDD_ace_arsenal) closeDisplay 0}] call CBA_fnc_execNextFrame;
+if (!isNull GVAR(currentBox) && {GVAR(currentBox) isEqualTo _object} && {isNil QGVAR(ignoredVirtualItems)}) then {
```

Comment:
I think this can be called in preInit, so I would stick with the `!isNil` check.